packages/run-protocol/src/econ-behaviors.js

```diff
@@ -74,7 +73,7 @@ export const startEconomicCommittee = async (
   },
   electorateTerms = {
     committeeName: 'Initial Economic Committee',
-    committeeSize: 1,
+    committeeSize: 3,
```

Comment:
This apparently causes a failure at https://github.com/Agoric/agoric-sdk/runs/6167083776?check_suite_focus=true#step:5:9111